docs/concepts/05_credentials/02_ctypes.md

```diff
@@ -19,10 +19,37 @@ This data format is used to define [CType models](https://github.com/KILTprotoco
 The following are all required properties of the schema, with no additional properties allowed:
 
 - **Identifier**: `$id` in the format `kilt:ctype:0x{cTypeHash}`.
-- **KILT specific JSON-Schema**: Accessible at [http://kilt-protocol.org/draft-01/ctype#](http://kilt-protocol.org/draft-01/ctype#).
+- **KILT specific JSON-Schema**: Accessible at [ipfs://bafybeiah66wbkhqbqn7idkostj2iqyan2tstc4tpqt65udlhimd7hcxjyq/](ipfs://bafybeiah66wbkhqbqn7idkostj2iqyan2tstc4tpqt65udlhimd7hcxjyq/).
 - **Title**: Defines a user-friendly name for the CType that makes it easier for users to contextualize.
 - **Properties**: Set of fields (e.g., name, birthdate) that the CType can contain, and hence that the Claimer can have attested.
 
+:::warning
+ Deprecation Warning: Property Swap
+
+Warning: The following CType Schema property value is deprecated.
+Please update your code accordingly to avoid any issues.
+It will mean you should update existing CTypes.
+The previous version of the CType will work, however, it is advised to upgrade.
+
+Old Property Value:  `"$schema": "http://kilt-protocol.org/draft-01/ctype#"`
+
+New Property Value:  `"$schema": "ipfs://bafybeiah66wbkhqbqn7idkostj2iqyan2tstc4tpqt65udlhimd7hcxjyq/"`
```

Comment:
```suggestion
 Deprecation Warning: CType metaschema draft-01

CTypes implementing the CType metaschema with id `http://kilt-protocol.org/draft-01/ctype#` do not prevent faulty or malicious attester implementations from adding additional, unexpected properties to a claim.
This metaschema is therefore considered deprecated and is not recommended to be used in the creation of new CTypes.

Please make sure to use sdk version **0.33.0** or newer when creating CTypes, which defaults to using the new metaschema  `ipfs://bafybeiah66wbkhqbqn7idkostj2iqyan2tstc4tpqt65udlhimd7hcxjyq/`.
```

Reply:
The second paragraph doesn't make sense.

```suggestion
Deprecation Warning: CType metaschema draft-01

Warning: `Claims` with the `CType` metaschema, identified as `http://kilt-protocol.org/draft-01/ctype#`, may lack prevention against faulty or malicious attester implementations, allowing them to add unexpected properties to the claim.
Please update your code accordingly to avoid any issues.

Old `$schema` value:  "http://kilt-protocol.org/draft-01/ctype#"

New `$schema` value:  "ipfs://bafybeiah66wbkhqbqn7idkostj2iqyan2tstc4tpqt65udlhimd7hcxjyq/"

The old metaschema is therefore considered deprecated and is not recommended to be used in the creation of new CTypes.

Please make sure to use SDK version **0.33.0** or newer when creating CTypes, which defaults to using the new metaschema  `ipfs://bafybeiah66wbkhqbqn7idkostj2iqyan2tstc4tpqt65udlhimd7hcxjyq/`.
```

does this read better?

I think it is essential to include the values so people know explicitly what they are and what they look like. Otherwise, they may miss it.

Reply:
I had GPT4 fix the language, which reads like this: 

> Deprecation Warning: CType Metaschema Draft-01
CTypes based on the `[http://kilt-protocol.org/draft-01/ctype#`](http://kilt-protocol.org/draft-01/ctype%23%60) metaschema are susceptible to faulty or malicious attester integrations that may introduce unexpected properties to a claim. Due to this vulnerability, this version of the metaschema is deprecated and its use is discouraged in the creation of new CTypes.
For optimal security and functionality, it is recommended to use SDK version 0.33.0 or later for creating CTypes. This newer version defaults to using the updated metaschema available at `[ipfs://bafybeiah66wbkhqbqn7idkostj2iqyan2tstc4tpqt65udlhimd7hcxjyq/`](ipfs://bafybeiah66wbkhqbqn7idkostj2iqyan2tstc4tpqt65udlhimd7hcxjyq/%60).

 I think this is pretty much the message we want to convey: do not produce more legacy CTypes (and move to the new version at your earliest convenience, but that comes below)

Reply:
As per Majella:

> (ChatGPT answer)

> CTypes based on the [http://kilt-protocol.org/draft-01/ctype#](http://kilt-protocol.org/draft-01/ctype%23%60) metaschema are susceptible to faulty or malicious attester integrations that may introduce unexpected properties to a claim. Due to this vulnerability, this version of the metaschema is deprecated and its use is discouraged in the creation of new CTypes.
> For optimal security and functionality, it is recommended to use SDK version 0.33.0 or later for creating CTypes. This newer version defaults to using the updated metaschema available at [ipfs://bafybeiah66wbkhqbqn7idkostj2iqyan2tstc4tpqt65udlhimd7hcxjyq/](ipfs://bafybeiah66wbkhqbqn7idkostj2iqyan2tstc4tpqt65udlhimd7hcxjyq/%60).
>

**Add:**

This also means you should update existing CTypes.

The previous version / existing CTypes will work in the short term. However, it is advised to upgrade as soon as possible.

Reply:
Is this language we should add verbatim? If not, how about something along the lines of 
> While existing CTypes will continue to work in the short term, we advise to upgrade to the latest metaschema at your earliest convenience.

Reply:
I think its fine to have either. I will add your suggestion.